viz_scripts/plots.py

```diff
@@ -211,32 +176,12 @@ def energy_impact(x,y,color,plot_title,file_name):
     plt.legend(labels=objects, handles=patches, loc='upper right', borderaxespad=0, fontsize=15, frameon=True)
     plt.savefig(SAVE_DIR+file_name+".png", bbox_inches='tight')
 
-def barplot_mode(data,x,y,plot_title,file_name):
-    all_labels= ['Gas Car, drove alone',
-                 'Bus', 
-                 'Train', 
-                 'Free Shuttle',
-                 'Taxi/Uber/Lyft', 
-                 'Gas Car, with others', 
-                 'Bikeshare',
-                 'Scooter share',
-                 'E-bike', 
-                 'Walk', 
-                 'Skate board', 
-                 'Regular Bike', 
-                 'Not a Trip',
-                 'No Travel', 
-                 'Same Mode', 
-                 'E-car, drove alone',
-                 'E-car, with others',
-                 'Air',
-                 'Other']
-    
-    colours = dict(zip(all_labels, plt.cm.tab20.colors[:len(all_labels)]))
+def barplot_mode(data,x,y,plot_title, labels, file_name):
+    colors = dict(zip(labels, plt.cm.tab20.colors[:len(labels)]))
     sns.set(font_scale=1.5)
     f = plt.subplots(figsize=(15, 6))
     sns.set(style='whitegrid')
-    ax = sns.barplot(x=x, y=y, palette=colours,data=data, ci=None)
+    ax = sns.barplot(x=x, y=y, palette=colors,data=data, ci=None)
```

Comment:
Ditto

Reply:
Reverted the changes. Reverted it to colours instead of colors.